mappings/net/minecraft/client/option/Option.mapping

```diff
@@ -58,6 +58,8 @@ CLASS net/minecraft/class_316 net/minecraft/client/option/Option
 	FIELD field_26925 HIDE_MATCHED_NAMES_TOOLTIP Lnet/minecraft/class_2561;
 	FIELD field_27956 TOGGLE_TEXT Lnet/minecraft/class_2561;
 	FIELD field_27957 HOLD_TEXT Lnet/minecraft/class_2561;
+	FIELD field_32146 MONOCHROME_LOGO Lnet/minecraft/class_4064;
```

Comment:
Can you map the lambda method parameters here?

Reply:
Do you mean that? I'm not sure but for me, it is already mapped

Reply:
![image](https://user-images.githubusercontent.com/38355282/113608887-68ce5b80-964b-11eb-994f-8aa18395b038.png)

Reply:
All four of the lambda parameters on the line are unmapped. In the Enigma editor launched through `./gradlew yarn`, they will be red.

Reply:
Oh okay. This is the first time I'm doing something with mappings, I don't know how to do that

Reply:
You just open a command line interface like how you open one for mod dev (at the root directory for yarn), and run `./gradlew yarnU` to open yarn with unpicked constants. Then you go to that class, and some lambda parameters will be mappable. You can rename there.

On a side note, I don't think it's worth requesting a change for lambda params, as they aren't really used outside of enigma; mapped source doesn't care, javadoc can't render them, and they are highly susceptible to code changes (lost across updates very easily)